djangocms_versioning/admin.py

```diff
@@ -99,18 +101,35 @@ def content_link(self, obj):
     content_link.admin_order_field = 'content'
 
     def grouper_form_view(self, request):
+        """Displays an intermediary page to select a grouper object
+        to show versions of.
+        """
         context = dict(
             self.admin_site.each_context(request),
             opts=self.model._meta,
-            form=grouper_form_factory(self.model._content_model)(),
+            form=grouper_form_factory(self.model._source_model)(),
         )
         return render(request, 'djangocms_versioning/admin/grouper_form.html', context)
 
     def changelist_view(self, request, extra_context=None):
         if not request.GET:
-            return self.grouper_form_view(request)
+            opts = self.model._meta
```

Comment:
A comment on what's happening here would help readability. Something like "We expect to have a grouper id in GET params. If we don't then redirect to make the user choose a grouper"